public/scripts/index.js

```diff
@@ -1,138 +1,162 @@
-    }
-    if (numValue === 3 && currentFrame.length >= 17 && currentFrame[16][16] != undefined){
-        clear();
-        drawLWSS(currentFrame, context, cellSize)
-    }
-
-    if (currentFrame === undefined){
-        alert('you must draw a grid')
-    }
-    else if (numValue === 0){
-        alert('You must select a preset');
-    }
-    else if (currentFrame.length < 17){
-        alert('Grid is too small. The minimum size is 17x17');
-    }    
+function drawPreset () {
+  let Nwid = (Math.ceil(window.innerWidth / 8) * 8)
+  let Nhei = (Math.ceil(window.innerHeight / 8) * 8)
+  let wid = Nwid - 32
+  let hei = Nhei - 32
+  const arrH = hei / cellSize
+  const arrW = wid / cellSize
+  currentFrame = makeMatrix(arrH, arrW)
+  const value = getPreset()
+  const numValue = parseInt(value)
+  if (numValue === 1 && currentFrame.length >= 17 && currentFrame[16][16] !== undefined) {
+    console.log(currentFrame)
+    clear()
+    drawToad(currentFrame, context, cellSize)
+  }
+  if (numValue === 2 && currentFrame.length >= 17 && currentFrame[16][16] !== undefined) {
+    clear()
+    drawAcorn(currentFrame, context, cellSize)
+  }
+  if (numValue === 3 && currentFrame.length >= 17 && currentFrame[16][16] !== undefined) {
+    clear()
+    drawLWSS(currentFrame, context, cellSize)
+  }
+
+  if (currentFrame === undefined) {
+    alert('you must draw a grid')
+  } else if (numValue === 0) {
+    alert('You must select a preset')
+  } else if (currentFrame.length < 17) {
+    alert('Grid is too small. The minimum size is 17x17')
+  }
 }
 
-function clear(){
-    let rows = document.getElementById("rows").value;
-    let cols = document.getElementById("columns").value;
-    currentFrame = makeMatrix(rows, cols);
-    intermediateFrame = makeMatrix(rows, cols);
-    nextFrame = makeMatrix(rows, cols);
-    draw(currentFrame, context, cellSize);
+function clear () {
+  let Nwid = (Math.ceil(window.innerWidth / 8) * 8)
+  let Nhei = (Math.ceil(window.innerHeight / 8) * 8)
+  let wid = Nwid - 32
+  let hei = Nhei - 32
+  const rows = hei / cellSize
+  const cols = wid / cellSize
+  currentFrame = makeMatrix(rows, cols)
+  intermediateFrame = makeMatrix(rows, cols)
+  nextFrame = makeMatrix(rows, cols)
+  draw(currentFrame, context, cellSize)
 }
-function animate(){
-    let checkedVal = document.getElementById('animate').checked;
-    console.log(checkedVal);
-    if(checkedVal === true){
-        setTimeout(function(){ simulate(); }, 500);
-        setTimeout(function(){ animate(); }, 500);
-    }
-    else{
-        console.log('NOT ANIMATING');
-    }
+function animate () {
+  const checkedVal = document.getElementById('animate').checked
+  console.log(checkedVal)
+  if (checkedVal === true) {
+    setTimeout(function () { simulate() }, 500)
+    setTimeout(function () { animate() }, 500)
+  } else {
+    console.log('NOT ANIMATING')
+  }
 }
 
-function alert(){
-    console.log('ALERT');
-}
+function resize () {
```

Comment:
Actually the whole UI becomes slightly larger than the viewport because when you set the heigt of the matrix to the `window.innerHeight` you don't account for the height taken up by the buttons at the top. You should subtract the height of the buttons to get the perfect fit for the grid.